utils/db/seedDB.js

```diff
@@ -2,10 +2,18 @@ const faker = require('faker');
 
 const db = require('../../server/database');
 
+const listings = [];
+
 function generateReview() {
+  let count = 0;
+  while (count < 4) {
+    listings.push(faker.random.uuid());
```

Comment:
@gorakong The listing ids need to be consistent across all of the services. Please use a random value between 1 and 100.

Reply:
@jcelder Sounds good. I have made the appropriate changes.